bundles/af-core/src/test/resources/form/recaptcha/exporter-recaptcha.json

```diff
@@ -1,25 +1,32 @@
 {
-    "id": "recaptcha-4b672680a2",
-    "fieldType": "captcha",
-    "name": "test-recaptcha",
-    "visible": true,
-    "type": "string",
-    "enabled": true,
-    "readOnly": false,
-    "properties": {
-      "fd:dor": {
-        "dorExclusion": false
-      },
-      "fd:path": "/content/recaptcha"
-    },
-    "label": {
-      "value": "CAPTCHA",
-      "visible": true
+  "id": "recaptcha-4b672680a2",
+  "fieldType": "captcha",
+  "name": "test-recaptcha",
+  "visible": true,
+  "type": "string",
+  "enabled": true,
+  "readOnly": false,
+  "properties": {
+    "fd:dor": {
+      "dorExclusion": false
     },
-    "events": {
-      "custom:setProperty": [
-        "$event.payload"
-      ]
-    },
-    ":type": "core/fd/components/form/recaptcha/v1/recaptcha"
+    "fd:path": "/content/recaptcha",
+    "fd:captcha": {
+      "siteKey": null,
```

Comment:
To make the fd:captcha properties more generic and adaptable to different captcha implementations, you can define a more flexible structure, something like this,

```
"fd:captcha": {
  "provider": "recaptcha",
  "config": {
    "siteKey": null,
    "uri": "https://www.recaptcha.net/recaptcha/api.js",
    "size": "compact",
    "theme": "light",
    "type": "image"
  }
}
```

"provider" property is to specify the captcha provider (e.g., "recaptcha", etc.). The "config" property is an object that contains the provider-specific configuration properties.

You can move the provider specific changes to `AbstractCaptcha` class with an abstract method called getProvider() and the config property would be returned by its implementation